tools/mtmd/mtmd.cpp

```diff
@@ -517,52 +525,58 @@ struct mtmd_tokenizer {
 
                 // add overview image (first)
                 if (ctx->ov_img_first) {
-                    if (ctx->tok_ov_img_start != LLAMA_TOKEN_NULL) {
-                        add_text({ctx->tok_ov_img_start});
+                    if (!ctx->tok_ov_img_start.empty()) {
+                        add_text(ctx->tok_ov_img_start);
```

Comment:
maybe we can now simplify this by removing all of these `if` conditions. the case of empty vector already handed inside `add_text`